partner/pms-common/src/main/java/io/mosip/pms/common/util/AuthenticationContextRefUtil.java

```diff
@@ -86,8 +98,11 @@ private Map<List<String>, String> getAllClaims() {
 					new TypeReference<Map<String,Claims >>() {
 					});
 			Map<List<String>, String> claimsMap = new HashMap<>();
+			List<String> allowedClaims = getSupportedClaims();
 			for(Map.Entry<String,Claims> mapElement : map.entrySet()) {
```

Comment:
if we loop on the supported list of claims, instead of all the entries available in identity-mapping.json you avoid an "if condition" in the loop.

Reply:
Done the same as suggested.